src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -184,6 +189,30 @@ private String commandStringGenerator(
     return commandString.toString();
   }
 
+  /**
+   * Reads the docker server configuration from file ssh.rc under "/pathToResourceDocker".
+   */
+  private void readServerConfig() {
+    String configFilePath = pathToResourceDocker + "/ssh.rc";
+    try {
+      List<String> lines = Files.readAllLines(Paths.get(configFilePath), Charset.forName("UTF-8"));
+      for (String line : lines) {
+        if (line.contains("p2p_server=") && !line.endsWith("=")) {
+          setKeyServerIp(line.split("=")[1]);
+        } else if (line.contains("http_port=") && !line.endsWith("=")) {
+          setKeyServerHttpPort(line.split("=")[1]);
+        }
```

Comment:
I think I have now an idea how I could explain it:
How about using a simple "else"-statement where we put something in like 
>logger.log("No parameters found, please check file");